ip-messaging/tokens/initialization/initialization.java

```diff
@@ -0,0 +1,14 @@
+import com.twilio.chat.ChatClient;
+import com.twilio.chat.ChatClientListener;
+
+ChatClient.Properties.Builder builder = new ChatClient.Properties.Builder();
+ChatClient.Properties properties = builder.createProperties();
+// See below for event listener implementation
+ChatClientListener listener = new ChatClientListener();
```

Comment:
`ChatClientListener` is an interface, you can't create an instance.

Reply:
Use `Listener` probably, as this is what is used below in the examples.

Reply:
Ouch, facepalm